tutorials/index.md

```diff
@@ -127,7 +129,7 @@ Some guidance for how to access / install the tutorials:
   - After the installation is complete run cell “Git-clone helper files for tutorials”
   - There are many download data cells. Pick only the data cell for your tutorial and run that cell. This may take a few minutes depending on the size of the data.
   - You are now ready to execute your notebook.
-  - Open a new tab in Chrome
+  - Open a new tab in Chrome and go to colab.research.google.com
   - Click on File->Open Notebook
   - In the window click on the Github tab
   - Enter brainiak/tutorials
```

Comment:
This also needs to change to Enter `brainiak tutorials`

Reply:
Oh, I thought they were different on purpose. OK.